### PR TITLE
Loss calc

### PR DESCRIPTION
**Pull request by yhenon**
Divide by `1` when there is no assigned boxes, instead of dividing by `keras.backend.epsilon()`. Stabilises training significantly.

### CODE REVIEW

**Comment by hgaiser** [comment]
Looks good, thanks!
